tests/serde-se.rs

```diff
@@ -1,7 +1,7 @@
 use quick_xml::de::from_str;
 use quick_xml::se::Serializer;
 use quick_xml::utils::Bytes;
-use quick_xml::DeError;
+use quick_xml::SeError;
```

Comment:
For consistency, this also could be imported as `quick_xml::se::SeError` in line 2